exec/plugin-commands-script-runners/src/run.ts

```diff
@@ -43,13 +45,22 @@ export const RESUME_FROM_OPTION_HELP = {
   name: '--resume-from',
 }
 
+export const SEQUENTIAL_OPTION_HELP = {
+  description: 'Run a multiple scripts specified with workspace selector or a wildcard script selector sequential.\
```

Comment:
The description of this option is confusing because we have a parallel option but this is unrelated to that. I'd just write that scripts are executed one at a time.

Reply:
Indeed. I updated help message more simple 3864f03